src/__tests__/message.model.test.ts

```diff
@@ -0,0 +1,45 @@
+import { Guid } from "guid-typescript";
+import { Message } from "../models";
+import { DateManipulator } from "../util/dateManipulator";
+
+let type: string;
+let originalSenderGuid: string;
+let body: string;
+let correlationId: string;
+let message: Message;
+let message2: Message;
+
+beforeAll(() => {
+    type = "ping";
+    originalSenderGuid = Guid.create().toString();
+    body = "body";
```

Comment:
If this variable doesn't change, please consider making it a constant.

Example (outside of the beforeAll() method):
const body = "ping";